### DISCUSSION TITLE
FDS-EVAC - Exit Counter

### DISCUSSION THREAD

**Discussion by joseragonfid** [firemodels/fds#11947] [Announcements]
Good afternoon,

I have added an EXIT with COUNTER ONLY in a transit zone. The problem is, as attached to the photo, that when there are crowds, there are quite a few people who count them more than once, thus giving me false numbers.

![image](https://github.com/firemodels/fds/assets/138492057/1c512256-b155-4880-a0d2-0f379f25fd11)

How could this be solved and only count one person once.

Greetings and thanks

**Comment by gjaeger** [comment]
@joseragonfid You can use `DOOR`->`ENTR` instead of `&EXIT ... COUNTER ONLY=.TRUE.`

This is not an issue, rather a question about an old FDS version with the FDS+Evac module.
